client/admin/app-bar-title.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import { AppBarTitle } from '../app-bar/app-bar.jsx'
+
+@connect(state => ({ routing: state.routing }))
+export default class AdminTitle extends React.Component {
+  render() {
+    const { routing: { location: { pathname } } } = this.props
+
+    let appBarTitle
+    if (pathname === '/admin') {
```

Comment:
You've probably already thought about this, but it'd be really nice if this could be specified on the route entries in some way